crates/attestation/Cargo.toml

```diff
@@ -1,23 +1,35 @@
 [package]
-name = "cryptography"
+name = "attestation"
 version = "0.2.0"
 edition = "2021"
 license = "AGPL-3.0"
-description = "Cryptography library for Steward"
+description = "Common workload attestation validation library"
+
+[features]
```

Comment:
We most probably want `sgx` and `snp` enabled by default

Reply:
Done